README.md

```diff
@@ -1,67 +1,99 @@
 
-![example_use_in_variables](https://user-images.githubusercontent.com/38699812/89090700-42447c00-d36a-11ea-8148-84af7cddfa21.PNG)
+## Heroku CI integration
+The Buildpack can be easily integrated with Heroku's Continuous Integration service ([Heroku CI](https://devcenter.heroku.com/articles/heroku-ci)) to automatically run linters and tests. 
 
-<!-- TODO: ## 📌 LICENCE -->
-<!-- TODO: ## 📌 CONTRIBUTE -->
+Use the following steps:
 
-## 📝 Final note
-   This buildpack is unofficial that means i don't have any conection with Heroku or Google from Flutter developer team <!--Although I would like belonging to any of the two :D -->. This repository was made as a hobby, i'm always interested in learn new things, this is one demostration of that :)
+1. Create a Heroku [pipleine](https://devcenter.heroku.com/articles/pipelines)
+2. Enable Heroku CI in the pipeline's configuration
+3. Create an `app.json` file in the root of your repository
+
+The `app.json` file should contain the app configuration, including the testing command that should be run by Heroku CI. All `app.json` options can be found in [Heroku's `app.json` documentation](https://devcenter.heroku.com/articles/app-json-schema). What follows is a very minimalistic (but working) `app.json` that runs Dart's built-in analyzer and your project's Flutter tests:
+
+```
+{
+  "environments": {
+    "test": {
+      "scripts": {
+        "test": "flutter analyze && flutter test"
+      }
+    }
+  },
+  "buildpacks": [
+    {
+      "url": "https://github.com/ludwiktrammer/heroku-buildpack-flutter"
+    }
+  ],
+}
+```
+
+Alternatively, for repositories where the Flutter project is not located in the root of the directory, the following configuration can be used (where `flutter-directory` is the location of the Flutter project):
+
+```
+{
+  "environments": {
+    "test": {
+      "scripts": {
+        "test": "cd flutter-directory && flutter analyze && flutter test"
+      }
+    }
+  },
+  "buildpacks": [
+    {
+      "url": "https://github.com/ludwiktrammer/heroku-buildpack-flutter"
+    }
+  ],
+  "env": {
+    "FLUTTER_SOURCE_DIR": "flutter-directory"
+  }
+}
+```
+
+During tests, all your project's dependencies are present (including the Dart SDK, Flutter SDK, pub.dev requirements, and command-line tools), so you can use them when specifying the testing command. 
+
+#### Options
+
+Optionally, you can use [config variables](https://devcenter.heroku.com/articles/config-vars) to customize the Buildpack's behavior:
+
+| Variable |  Default        |  Description
+|----------|------------------| -------------------|
+| FLUTTER_CLEANUP | `true` | Whether to emoves sources, tools, and dependencies before deployment |
+| FLUTTER_VERSION | *Last version in [beta channel](https://flutter.dev/docs/development/tools/sdk/releases?tab=linux).* | The **name of the version** used to compile the project
```

Comment:
how does FLUTTER_VERSION affect actual version used? it looks that we always fetch latest version from git repo

or is it used only for verification that we are using version we want to use? if so then it's not clearly described

but I guess it can be merged like that - this is not a change introduced in this PR

Reply:
On line 79 in the `compile` a `flutter version $FLUTTER_VERSION`  command is executed, switching the version.